### ISSUE TITLE
Overlaped grenade title text

### ISSUE THREAD

**Issue by EstevaoCalera** [ReneRebsdorf/CS2-annotations#23]
This happens for all grenades that share the same spot.
![image](https://github.com/user-attachments/assets/9063bc0a-d0aa-4d52-a5b1-6de86e5eb84a)

It can be solved simply by changing the offset as the follow:
![image](https://github.com/user-attachments/assets/13e8b48a-efde-4acf-afd6-5d7588b47b50)

I recommend adding a test to check for overlapping text either

**Comment by ReneRebsdorf** [comment]
Yeah I agree, this is actually mentioned in #8 
Perhaps @JanElholm will help here :-)

As I mention in the other issue, we need to "round down" the accuracy of the positions, as otherwise they might not be found as overlapping.

I think we should keep this issue, as it is a better description than that in #8

**Comment by EstevaoCalera** [comment]
I am sorry about this duplication. Feel free to delete it if you want to.

This "round down" matter is an actually very good approach!

**Comment by ReneRebsdorf** [comment]
I have refactored the tests in pytest with #8 

I did make a bit of a shortcut which could be improved, some of the tests are done as part of the 'discovery', where if an assert fails, it will quit the rest of testing.
We should probably improve upon that.

Anyway, the structure and other tests are otherwise ready and merged, I'll ask @JanElholm (colleague) at the next opportunity for tests of these overlaps, and use that test to find the sinners.
